doc/devel/gitwash/matplotlib_for_dev.rst

```diff
@@ -0,0 +1,17 @@
+.. _matplotlib-for-dev:
+
+===================================================
+    Install for matplotlib source for development
+===================================================
+
+After obtaining a local copy of the matpotlib source code (:ref:`set-up-fork`),
+navigate to the matplotlib directory and run the following in the shell:
+
+::
+    
+    python setup.py develop
+
+This installs matplotlib for development (i.e., builds everything and places the
+symbolic links back to the source code).
```

Comment:
As noted in ` faq/installing_faq.rst` this command has to be called everytime that the c files change